WordPress/wordpress-playground
# Website: Add error report modal

This PR is missing data submission and shouldn't be merged.

## What is this PR doing?

When a PHP request fails a modal will load that allows users to report errors to us.

## What problem is it solving?

It will help us collect Playground errors that we need to improve stability. 

## How is the problem addressed?

By allowing users to submit error reports to us.

## Testing Instructions

- Checkout this branch
- Start the local dev server `npm run dev`
- Update [this line ](https://github.com/WordPress/wordpress-playground/pull/1102/files#diff-46008bd779d2dd5cb639272c824aa9529721d171731bc22c6b073c0d73aab3c0R9)to be true by default
- Confirm that a modal is opened
- Modify the input data
- Click _Report error_
- See the[ submitted report in Slack](https://wordpress.slack.com/archives/C06Q5DCKZ3L)

![Screenshot from 2024-03-22 10-42-32](https://github.com/WordPress/wordpress-playground/assets/1199991/d39c6acc-5694-4e8e-babc-f88aa90748ad)
